tools/tsan_playground/api_c11_acq_rel_barriers.h

```diff
@@ -0,0 +1,36 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+#pragma once
+
+#include <stdatomic.h>
+#include <stdint.h>
+#include <stdio.h>
+
+typedef atomic_uint_fast64_t atomic_t;
+typedef uint64_t value_t;
+
+#define ATOMIC_DEFINE(var, value) static atomic_t var = (value)
+
+void atomic_store_release(atomic_t* var, value_t value);
+value_t atomic_load_acquire(atomic_t* var);
+const char *get_mode(void);
```

Comment:
Should we create a common header file for all the values + function definitions?